### PR TITLE
Drop Rafter related configuration

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Remove test configuration and other periodic jobs related to rafter testing

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/13155

### CODE REVIEW

File: development/test-log-collector/pkg/config/config_test.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func TestLoadDispatchingConfig(t *testing.T) {
   channelID: "chanID1"
   onlyReportFailure: false
   testCases:
-    - "rafter"
+    - "busola-migrator"
```
Comment (moelsayed) [inline development/test-log-collector/pkg/config/config_test.go:135]: Why are we replacing this?

File: development/tools/jobs/kyma/tests_test.go [hunk 2]
```diff
@@ -14,7 +14,37 @@ var tests = []struct {
...
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("generic"),
+		},
+	},
+	{path: "connector-service-tests", image: tester.ImageGolangBuildpack1_16, suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("generic"),
+		},
+	},
+	{path: "integration/api-gateway", image: tester.ImageGolangBuildpack1_16, suite: tester.NewGenericComponentSuite,
```
Comment (moelsayed) [inline development/tools/jobs/kyma/tests_test.go:47]: Can please take a look at these? I think there might be rebase errors there...

**Review by moelsayed — COMMENTED** [review]
looks good, just a couple of notes..

File: development/test-log-collector/pkg/config/config_test.go (hunk 1 above)
Comment (kwiatekus) [reply development/test-log-collector/pkg/config/config_test.go:135]: well.. this is just a config test that was mentioning rafter as an example..
If the config for rafter is being removed I thought its not a good case for this test case so I replaced it with sth else

**Review by grego952 — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Review by Halamix2 — APPROVED** [review]

**Pull request merged**